sql/dql/README.md

```diff
@@ -22,8 +22,8 @@ standards:
     objectives:
       0: Connect any database client to a database server on a local host
       1: Connect any database client to a database server on a remote host
-      2: Break down connection URIs into their component elements (protocol, hostname, credentials, port, database name, etc)
-      3: Assemble connection URIs from component elements
+      2: Break down connection URLs into their component elements (protocol, hostname, credentials, port, database name, etc)
```

Comment:
Not important since standards aren't used (in fact you can just nuke them) but URI is perfectly valid here.

> While they are used interchangeably, there are some subtle differences. For starters, URI stands for uniform resource identifier and URL stands for uniform resource locator. ... You see, a URI can be a name, locator, or both for an online resource where a URL is just the locator.

Reply:
ooo nice, I actually had no idea. the more you know